be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -320,10 +320,9 @@ Status BetaRowsetWriter::_create_segment_writer(
     }
 
     DCHECK(wblock != nullptr);
```

Comment:
need not change this code, it does not matter whether it is a local variable or a member variable.